### ISSUE TITLE
When the dkron agent panics, the plugin processes attached to it are not automatically recycled

### ISSUE THREAD

**Issue by jexbow** [distribworks/dkron#1496]
**Describe the bug**
When the dkron agent panics, the plugin processes attached to it are not automatically recycled

**Comment by vcastellm** [comment]
Yes this is how the plugin system works, it's a tradeoff, and we can try to find several non-elegant solutions but the best option is to minimize or remove the possibility of panics, that should be well covered at this point.

Closing this for now but let us know any panic you experienced.
